uportal-war/src/main/java/org/springframework/orm/jpa/JpaAccessor.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2002-2012 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.orm.jpa;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceException;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.dao.support.DataAccessUtils;
+
+/**
+ * Base class for JpaTemplate and JpaInterceptor, defining common
+ * properties such as EntityManagerFactory and flushing behavior.
+ *
+ * <p>Not intended to be used directly.
+ * See {@link JpaTemplate} and {@link JpaInterceptor}.
+ *
+ * @author Juergen Hoeller
+ * @since 2.0
+ * @see #setEntityManagerFactory
+ * @see #setEntityManager
+ * @see #setJpaDialect
+ * @see #setFlushEager
+ * @see JpaTemplate
+ * @see JpaInterceptor
+ * @see JpaDialect
+ * @deprecated as of Spring 3.1, in favor of native EntityManager usage
+ * (typically obtained through {@code @PersistenceContext})
```

Comment:
Agreed.  At this point it is a time constraint.  Gary is tied up on a project with scrum deadlines to deal with (you know how it goes).